packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -27,13 +29,15 @@ Collectively, this is known as hydration. All current generations of frameworks
 
 Qwik is different because it does not require hydration to resume an application on the client. Not requiring hydration is what makes the Qwik application startup instantaneous.
 
-All other frameworks' hydration **replays** all the application logic in the client. Qwik instead pauses execution in the server, and resumes execution in the client.
+All other frameworks' hydration **replays** all the application logic in the client. Qwik instead pauses execution in the server , and resumes execution in the client.
+[//]: # (The very consept of pausing the application is huge, a schema or a quick explaination could help making a mental model.)
```

Comment:
agree

I have something I prepared for the course
I can add it after I'm done with the first version of the course